modules/logging-upgrading-loki.adoc

```diff
@@ -26,6 +26,41 @@ To update the {loki-op} to a new major release version, you must modify the upda
 
 . In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-5.y*, and click *Save*. Note the `loki-operator.v5.y.z` version.
 
+. Check if the `LokiStack` custom resource contains the `v13` schema version and add it if it is missing:
++
+[IMPORTANT]
+====
+You must add the `v13` schema version because it is required for the {clo} 5.9 or later.
```

Comment:
I'm not sure if "required" is the correct wording here. Everything in 5.9 also works with v12, we just recommend switching to 13, because performance should be better.
It is required for future features (OTLP support), which is available in 6.0/6.1 as a tech preview.

Reply:
> Everything in 5.9 also works with v12

Should we ignore the **LokistackSchemaUpgradesRequired** alert in the web console?

@r2d2rnd, to double-check, is 5.9 an affected version?

Reply:
No, the alert exists in the earlier versions already. It's just a warning though and the wording also says "it is recommended to update" (fwiw: just noticed that we might have a spelling error in that description ...)

The alert was added to "gently nudge" people to update to newer schema versions. Maybe putting "required" in the name wasn't gentle enough.

Reply:
@xperimental and @r2d2rnd,

FYI, in the screenshot shown in https://access.redhat.com/solutions/7063482, if the alert details are still the same for both Logging 5.9 and 6, I noticed a combination of the alert severity **Warning** and alert title ending with `Required` and the alert's **Summary** text containing `outdated`. If so, there might be a gap between the intention (gentle nudge) and the impact of the message wording (customers scared into action and opening support tickets).

I am not the docs owner for Logging, but I have a few suggestions for you:
1. For the customers who already opened support tickets, consider sharing the link to the new docs section `Upgrading .... schema` that merging this PR will create in the `docs.redhat.com` (not `docs.openshift.com`, which is planned for shutdown) after this PR is published.
2. If you can, add the link to the just mentioned new docs section `Upgrading .... schema` in this alert for the next release so that users can open the docs and complete the upgrade with minimum disturbance.
3. Update the schema-version alert texts for every Logging release.
4. Ensure sufficient emphasis and a call to action for customers to upgrade the Loki schema version in the release notes of the next and each affected release.
5. For every release in which the latest schema version increases, the docs content in the procedure in this PR has to be updated too.

Reply:
Thank you @max-cx for the suggestions. I'll try to update the article when this PR is published